fontforgeexe/charinfo.c

```diff
@@ -3251,8 +3251,11 @@ return( NULL );
 
     if ( ymax<=ICON_WIDTH ) ymax = ICON_WIDTH;
     if ( ymin>0 ) ymin = 0;
-    if ( xmax<xmin )
+    if ( xmax<xmin ) {
+        free(extras);
```

Comment:
Oups, I understand now. I forgot this part:
     for ( i=0; i<extracnt; ++i )
         BDFCharFree(extras[i]);